head/core/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+# This CMakeLists.txt handles compiling all the parts of the head
+# module that are portable between host and cross compilation as a static
+# library. It is included in both host and cross configurations.
+
+add_library(head-core STATIC
+        head.cpp
+        )
+
+set_target_properties(head-core
+  PROPERTIES CXX_STANDARD 20
+             CXX_STANDARD_REQUIRED TRUE)
+
+target_include_directories(head-core
+  PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/../include)
```

Comment:
This is also the wrong path for the include directory -- if you copied from gantry that project hasn't been updated in awhile. The path specified here would point to an include directory inside of the `head` subproject.

```
target_include_directories(head
        PUBLIC ${CMAKE_SOURCE_DIR}/include)
```

Reply:
So there is an include directory at root level, with sub directories for can, pipettes etc in it. Should we get rid of that and each subproject would have an include of their own nested inside of it? I figured the idea was for all subprojects to use a common include..

Reply:
thanks, finally makes sense to me. [0230e63](https://github.com/Opentrons/ot3-firmware/pull/104/commits/0230e63cf179970f6c151b24b1296bbe0661a901)